libs/core/algorithms/include/hpx/algorithms/traits/is_relocatable.hpp

```diff
@@ -0,0 +1,23 @@
+//  Copyright (c) 2023 Hartmut Kaiser
+//
+//  SPDX-License-Identifier: BSL-1.0
+//  Distributed under the Boost Software License, Version 1.0. (See accompanying
+//  file LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+#pragma once
+
+#include <utility>
```

Comment:
```suggestion
#include <type_traits>
```
Both, `is_destructible` and `is_move_constructible` are defined in `<type_traits>`

Reply:
Changed it